Users/Tutorial.rst

```diff
@@ -43,7 +43,7 @@ Command Line Interface
 
 In order to specify the files to analyze, you can use the ``--files``
 argument of coala like demonstrated below. For all file paths, you can
-specify (recursive) globs.
+specify (recursive) `globs <../Users/Glob_Patterns.html>`__.
```

Comment:
Please refer the address to http://docs.coala.io/en/latest/Users/Glob_Patterns.html .